rest/utilities_testing.go

```diff
@@ -883,8 +883,8 @@ func (cr ChangesResults) Summary() string {
 	return strings.Join(revs, ", ")
 }
 
-// RequireChangeRevVersion asserts that the given ChangeRev has the expected version for a given entry returned by _changes feed
-func RequireChangeRevVersion(t *testing.T, expected DocVersion, changeRev db.ChangeRev) {
+// RequireChangeRevVersion asserts that the given ChangeByVersionType has the expected version for a given entry returned by _changes feed
+func RequireChangeRevVersion(t *testing.T, expected DocVersion, changeRev db.ChangeByVersionType) {
 	RequireDocVersionEqual(t, expected, DocVersion{RevTreeID: changeRev["rev"]})
```

Comment:
Is this always supposed to assert on rev here, if so rename this function with revtree id, otherwise make the assertion more flexible for cv.